recipes/persim/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "persim" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 07b14bd7e17b187c760297954b55788396b6be31e1e9ead35e3d73ebfa488771
+  
+build:
+  noarch: python
+  number: 0
+  script: package_version={{ version }} python -m pip install --no-deps --ignore-installed .           # [not win]
+  script: set "package_version={{ version }}" && python -m pip install --no-deps --ignore-installed .  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - cython
+    - numpy >=1.13
+    - scipy
+    - scikit-learn
+    - matplotlib
+    - hopcroftkarp
+
+test:
+  imports:
+    - persim
```

Comment:
If you want to run the tests you should also add a commands section like:
```
  commands:
    - py.test --pyargs package_name
```

Reply:
Thank you!  Is it normal to run tests during this process? We have Travis setup for tests anyways so this might be redundant?

Reply:
Provided they're relatively fast it can be nice to ensure conda's binaries work as expected though for pure Python packages just importing is often good enough. It's up to you.

If you decide not to, you should remove `pytest` from the test dependencies.

Reply:
I've removed tests. Thank you.